ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -88,7 +88,7 @@ public final class SQLiteNormalizedCache: NormalizedCache {
   }
 }
 
-private let serializedReferenceKey = "reference"
+private let serializedReferenceKey = "$reference"
```

Comment:
Is this really necessary? What distinguishes a reference field from any other field is that it's a dictionary (of the form `[reference: <key>]`), whereas other fields are primitive types.